test/cypress/integration/simplified-account/service-settings/api-keys/api-keys.cy.js

```diff
@@ -82,13 +82,13 @@ describe('Settings - API keys', () => {
               cy.get('.govuk-summary-list__row').eq(2)
                 .within(() => {
                   cy.get('.govuk-summary-list__key').should('contain', 'Last used')
-                  cy.get('.govuk-summary-list__value').should('contain', token.lastUsed || '')
+                  cy.get('.govuk-summary-list__value').should('contain', token.lastUsed || 'Not used yet')
```

Comment:
I think this a problematic test. Using `||` allows for multiple valid states. 

Can you amend to separate test cases for each scenario (used and unused)?